packages/react-union/src/decorators/withErrorBoundary/withErrorBoundary.js

```diff
@@ -15,7 +16,7 @@ const withErrorBoundary = WrappedComponent => {
 		}
 
 		render() {
-			const { name } = this.props.descriptor;
+			const name = path(['props', 'descriptor', 'name'], this);
```

Comment:
refactor to `const getName = path(['props', 'descriptor', 'name'])` and move it above the component. Here just call `getName(this)`

Reply:
Yeah, that's better. Thanks!

Reply:
just to make it bit quicker